src/main/resources/.data/carpet-redcraft-addons/scripts/barrier.sc

```diff
@@ -40,6 +40,8 @@ if((item_tuple = inventory_get(player, to)) && item_tuple:0 == 'barrier' || (ite
 );
 
 __on_explosion_outcome(pos, power, source, causer, mode, fire, blocks, entities) -> (
+print(source);
+if (system_info('world_carpet_rules'):'explosionNoBlockDamage' == 'true' || (system_info('world_gamerules'):'mobGriefing' == 'false' && source != 'Primed TNT'), return());
```

Comment:
Le barriere non vengono droppate quando l'esplosione viene generata da un giocatore se la gamerule mobGriefing è impostata a false.

Reply:
In che senso?? Il player crea esplosioni??